src/ServiceControl/Hosting/Host.cs

```diff
@@ -55,6 +55,7 @@ protected override void OnStop()
         protected override void Dispose(bool disposing)
         {
             OnStop();
+            bootstrapper?.Dispose();
```

Comment:
is this the right place to call this?  Do we also need to call bootstrapper Stop in OnPause or OnStop?